### PR TITLE
Add initContainers to ambassador

### PR DESCRIPTION
**Pull request by kyschouv**
#### What this PR does / why we need it:
Enables `initContainers` to run before ambassador to set up the environment. Specifically, this will enable loading certificates from an Azure KeyVault FlexVolume and converting them to the necessary format for ambassador, similar to [this example](https://github.com/Azure/kubernetes-keyvault-flexvol/blob/master/docs/traefik-tls-certificate.md#initcontainer). This also adds the ability to add additional pod labels to enable `aad-pod-identity`.

#### Special notes for your reviewer:
I've tested this with both an `initContainers` defined and with it left empty (to make sure it doesn't break any current behavior), and both run successfully.

#### Checklist
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by kflynn** [comment]
Looks reasonable here; /cc flydiverny for another pair of eyes.

/ok-to-test

**Comment by kflynn** [comment]
Looks like our bump to 0.60.3 stomped on your version bump -- sorry about that!

**Comment by kyschouv** [comment]
No problem, I'll just bump the versions again =).

**Comment by kyschouv** [comment]
Alright it should be ready to go.

**Comment by Flydiverny** [comment]
missed this will take a look tomorrow-ish

/assign

File: stable/ambassador/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.60.3
 description: A Helm chart for Datawire Ambassador
 name: ambassador
-version: 2.2.5
+version: 2.2.6
```
Comment (Flydiverny) [inline stable/ambassador/Chart.yaml]: new values -> 2.3.0

**Pull request merged**